bubblesub/ui/console.py

```diff
@@ -341,6 +341,7 @@ def __init__(self, api: Api, parent: QtWidgets.QWidget) -> None:
         super().__init__(parent)
 
         self._api = api
+        self.setObjectName("console")
```

Comment:
Is there a better place for this call within this constructor? It's stuck between setting two internal properties

Reply:
I've copied the other methods placements. Tell me the best place.